src/main/java/com/commercetools/emailprocessor/model/ProjectConfiguration.java

```diff
@@ -0,0 +1,48 @@
+package com.commercetools.emailprocessor.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ProjectConfiguration {
+    private static final Logger LOG = LoggerFactory.getLogger(ProjectConfiguration.class);
+    private List<TenantConfiguration> tenants;
+
+    public ProjectConfiguration() {
```

Comment:
now this empty constructor `ProjectConfiguration()` must be removed

Reply:
no, because if it is removed this exception occurred:

`com.fasterxml.jackson.databind.JsonMappingException: Can not construct instance of com.commercetools.emailprocessor.model.ProjectConfiguration: no suitable constructor found, can not deserialize from Object value (missing default constructor or creator, or perhaps need to add/enable type information?)
 at [Source: /Users/ahalberkamp/dev/src/commercetools-email-retry-processor/out/test/resources/validProjectConfiguration.json; line: 2, column: 3]`